edx_rest_framework_extensions/tests/test_middleware.py

```diff
@@ -0,0 +1,93 @@
+"""
+Unit tests for safe_endpoints Middleware.
+"""
+import ddt
+from itertools import product
+
+from django.test import TestCase, RequestFactory
+from rest_framework.authentication import SessionAuthentication
+from rest_framework_jwt.authentication import BaseJSONWebTokenAuthentication
+from rest_framework.decorators import api_view
+from rest_framework.views import APIView
+
+from ..middleware import EnsureJWTAuthSettingsMiddleware
+from ..permissions import JwtHasScope
+
+
+class SomeIncludedPermissionClass(object):
+    pass
+
+
+class SomeJwtAuthenticationSubclass(BaseJSONWebTokenAuthentication):
+    pass
+
+
+def some_auth_decorator(include_jwt_auth=True, include_required_perm=True):
```

Comment:
Nit: If you are relying on values being explicitly declared, you can drop the default values.  Unless you are using that as a pseudo-comment about type?

Reply:
I put in defaults since not all future callers need to pass in values for those optional parameters.